plugins/gatsby-theme-iterative-docs/src/utils/front/Slugger.ts

```diff
@@ -0,0 +1,43 @@
+class Slugger {
```

Comment:
The `Slugger` class is very well written! At a glance, I was able to easily tell what everything does!

Reply:
Thanks, also took https://github.com/Flet/github-slugger/blob/master/index.js for reference.